<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>510</y>
      <width>210</width>
      <height>137</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="timeStampDisplayCheckBox">
         <property name="text">
          <string>显示时间戳</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="sendInterval">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ms/次</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="TimerSendCheck">
         <property name="text">
          <string>定时发送</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hexDisplay">
         <property name="text">
          <string>hex显示</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hexSend">
         <property name="text">
          <string>hex发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>发送</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearRecvAreaButton">
         <property name="text">
          <string>清空接收区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearSendAreaButton">
         <property name="text">
          <string>清空发送区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearStatistic">
         <property name="text">
          <string>清空统计</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>721</width>
      <height>141</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;多行文本输入区&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0,0">
     <item>
      <widget class="QComboBox" name="comList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudrateList">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>9600</string>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>460800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshCom">
       <property name="text">
        <string>刷新串口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="comSwitch">
       <property name="text">
        <string>打开串口</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>941</width>
      <height>451</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;数据接收区&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>编码规则</string>
     </property>
     <addaction name="actionUTF8"/>
    </widget>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>回车样式</string>
     </property>
     <addaction name="action_winLikeEnter"/>
     <addaction name="action_unixLikeEnter"/>
    </widget>
    <addaction name="actionCOM_Config"/>
    <addaction name="menu_6"/>
    <addaction name="menu_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionDrawHelp"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSaveData"/>
    <addaction name="actionReadData"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionReadConfig"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionYTMode"/>
    <addaction name="actionXYMode"/>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
  <action name="actionCOM_Config">
   <property name="text">
    <string>串口设置</string>
   </property>
   <property name="toolTip">
    <string>串口设置</string>
   </property>
  </action>
  <action name="actionDrawHelp">
   <property name="text">
    <string>绘图帮助</string>
   </property>
   <property name="toolTip">
    <string>绘图帮助</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>保存数据</string>
   </property>
  </action>
  <action name="actionReadData">
   <property name="text">
    <string>读取数据</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>保存配置</string>
   </property>
  </action>
  <action name="actionReadConfig">
   <property name="text">
    <string>读取配置</string>
   </property>
  </action>
  <action name="actionYTMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>YT图模式</string>
   </property>
  </action>
  <action name="actionXYMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>XY图模式</string>
   </property>
  </action>
  <action name="actionUTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF8</string>
   </property>
  </action>
  <action name="action_winLikeEnter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>\r\n</string>
   </property>
  </action>
  <action name="action_unixLikeEnter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>\n</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
